File: modulo06_colecoes/bee_1168_led.py
def main():
    qtd_casos_teste = int(input())
    contador = 0

    while (contador < qtd_casos_teste):
        caso = int(input())
        caso_para_string = my_to_string(caso)
        qtd_leds = 0
        contador = contador + 1

        for caractere in caso_para_string:
            if caractere == '1':
                qtd_leds = qtd_leds + 2
            elif caractere == '2':
                qtd_leds = qtd_leds + 5
            elif caractere == '3':
                qtd_leds = qtd_leds + 5
            elif caractere == '4':
                qtd_leds = qtd_leds + 4
            elif caractere == '5':
                qtd_leds = qtd_leds + 5
            elif caractere == '6':
                qtd_leds = qtd_leds + 6
            elif caractere == '7':
                qtd_leds = qtd_leds + 3
            elif caractere == '8':
                qtd_leds = qtd_leds + 7
            elif caractere == '9':
                qtd_leds = qtd_leds + 6
            elif caractere == '0':
                qtd_leds = qtd_leds + 6
                
        mostra_texto(f'{qtd_leds} leds\n')

def my_to_string(numero):
    return str(numero)

def mostra_texto(mensagem):
    print(mensagem)

main()

File: bee_crowd_atividades/atividade_colecoes/bee_1168_led.js
import { question } from "readline-sync";

function main(){
    let casos_teste = get_number('Quantos casos de teste?\n')
    
    for (let i = 0; i < casos_teste; i++){
        let caso = get_number(`Caso ${i+1}: `)
        let caso_para_string = my_to_string(caso)
        let qtd_leds = 0

        for (let caracter of caso_para_string){
            if (caracter == '1'){
                qtd_leds = qtd_leds + 2
            } 
            if (caracter == '2'){
                qtd_leds = qtd_leds + 5
            }
            if (caracter == '3'){
                qtd_leds = qtd_leds + 5
            }
            if (caracter == '4'){
                qtd_leds = qtd_leds + 4
            }
            if (caracter == '5'){
                qtd_leds = qtd_leds + 5
            }
            if (caracter == '6'){
                qtd_leds = qtd_leds + 6
            }
            if (caracter == '7'){
                qtd_leds = qtd_leds + 3
            }
            if (caracter == '8'){
                qtd_leds = qtd_leds + 7
            }
            if (caracter == '9'){
                qtd_leds = qtd_leds + 6
            }
            if (caracter == '0'){
                qtd_leds = qtd_leds + 6
            }
        }
        mostra_texto(`${qtd_leds} leds\n`)
    }
}

function my_to_string(numero){
    return numero.toString()
}

function mostra_texto(mensagem){
    console.log(mensagem)
}

function get_number(mensagem){
    return Number(question(mensagem))
}

main()